CONTRIBUTING.md

```diff
@@ -152,21 +177,22 @@ We have three branches for shared development: `master`, `releases` and `next-ma
 We use the offical [style guide](https://kotlinlang.org/docs/reference/coding-conventions.html) from Kotlin which is enforced using [ktlint](https://github.com/pinterest/ktlint) and [detekt](https://github.com/detekt/detekt).
 
 ```sh
-# Call from root folder to check if code is compliant.
+# Call from packages folder to check if code is compliant.
+cd packages
```

Comment:
This is actually not correct. We do mean the root folder, i.e. `<path>/realm-kotlin`. This is because these commands also check the rest of the project, not just the library code inside `packages`

Reply:
Okay, seemed to work only from `packages` for me but will try to resolve that. Thanks!

Reply:
Calling it from the root requires that you have called `./gradlew publishAllPublicationsToTestRepository` first.